--- ../unified_unix_agent/ocsinventory-agent 	2011-05-08 19:23:10.000000000 +0200
+++ ocsinventory-agent	2011-07-29 00:15:15.000000000 +0200
@@ -18,6 +18,18 @@
 #USA.
 #
 
+BEGIN {
+    use File::Basename;
+    use Cwd qw(abs_path);
+
+    my $scriptname = basename(__FILE__);
+    my $path = abs_path($0);
+    $path =~ s/$scriptname//g;
+
+    unshift(@INC,$path."/lib/");
+}
+
+
 use strict;
 use warnings;
 
